### ISSUE TITLE
code-syntax-highlight plugin generates unexpected codeblock content 

### ISSUE THREAD

**Issue by h3l1o5** [nhn/tui.editor#2497]
## Describe the bug
if codeblock's language info is unrecognizable or empty, the plugin doesn't escape html entities before insert content into DOM 

## To Reproduce
1. Go to https://nhn.github.io/tui.editor/latest/tutorial-example08-editor-with-code-syntax-highlight-plugin
2. type   
\```
<a
\```
 
## Expected behavior
in the preview screen it should render
```
<a
```

## Screenshots
![screenshot](https://user-images.githubusercontent.com/17796759/168011148-b0aeec69-f233-42e2-9eab-3a68f713db5a.png)

## Desktop (please complete the following information):
 - OS: macOS Catalina 10.15.7
 - Browser: Chrome
 - Version: 100.0.4896.127

**Comment by jwlee1108** [comment]
@h3l1o5 
Let me check. But you can use it if you specify the type as `jsx` or `html`.

**Comment by h3l1o5** [comment]
@jwlee1108 Yes, that's one of our workaround right now. It's not critical but would be nice if the plugin could handle the default case (which is no language) properly.

**Comment by EngincanV** [comment]
Any updates?
